### ISSUE TITLE
Module path should be "github.com/Sandertv/mcwss", not "github.com/sandertv/mcwss"

### ISSUE THREAD

**Issue by KateGo520** [Sandertv/mcwss#13]
## Background
Module path is inconsistent with go import path.
GO111MODULE=on, as README said, run `go get github.com/Sandertv/mcwss`:

```
go: downloading github.com/Sandertv/mcwss v1.2.0
go: github.com/Sandertv/mcwss upgrade => v1.2.0
go get: github.com/Sandertv/mcwss@v1.2.0: parsing go.mod:
        module declares its path as: github.com/sandertv/mcwss
                but was required as: github.com/Sandertv/mcwss 
```

###  Fix the module path:
1) Rename the module path to "github.com/Sandertv/mcwss": https://github.com/Sandertv/mcwss/blob/master/go.mod#L1 
```
module github.com/Sandertv/mcwss
require (
	…
) 
```
2) Change the README document to use `go get github.com/sandertv/mcwss`.

**Comment by KateGo520** [comment]
@Sandertv  @Tsubashi  Could you help me review this issue? Thx :p

**Comment by Sandertv** [comment]
For backwards compatibility reasons I don't think renaming the module is an option. I'll change the README to use all lowercase.

**Comment by Sandertv** [comment]
Should be fixed in 0244b379697bf0021212c2587d6234eba26aef48, let me know if that is sufficient.

**Comment by KateGo520** [comment]
@Sandertv I think that's sufficient. Thank you for your reply.
